spec/features/api_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+
+RSpec.describe "Api" do
+  before :all do
+    drop_dummy_database
+    remove_project_directory
+    create_dummy_project("api" => true)
+  end
+
+  it "adds power_api related gems to Gemfile" do
```

Comment:
Example has too many expectations [6/5].

Reply:
aquí juntaría las gemas en un mismo include porque permite múltiples argumentos, osea:
`expect(gemfile_content).to include(a, b, c, d, e, f)` y así cumplimos con lo que nos pide el mono.

Reply:
ahora que saqué algunas gemas, quedaron menos.